### ISSUE TITLE
Add Commonwealth Electoral Divisions 2018 - ABS + AEC

### ISSUE THREAD

**Issue by stevage** [TerriaJS/terriajs#3203]
Boundary files available here: http://www.abs.gov.au/AUSSTATS/abs@.nsf/DetailsPage/1270.0.55.003July%202018?OpenDocument

* [x] Add catalog item to NationalMap
* [x] Generate and host vector tiles
* [x] Configure in region mapping
* [x] Update csv-geo-au

**Comment by AnaBelgun** [comment]
the latest AEC boundaries are here: https://www.aec.gov.au/Electorates/gis/gis_datadownload.htm

**Comment by stevage** [comment]
Turns out the AEC boundary files are a bit of a mess. For starters, every state is provided separately, with inconsistent file names and formats, and you have to carefully choose the "current" version of each boundary, rather than the "superseded" ones.

We can use 'Sortname' for as the boundary identifier, but need to deal with some case inconsistency for NSW.

Currently looking at an issue with missing boundaries in NSW and Tas.

**Comment by AnaBelgun** [comment]
yes, indeed @stevage that's why we stayed away from them for a while
if you need help clarifying anything let me know and I'll put you in touch with the boundaries data manager there

**Comment by stevage** [comment]
The PR addresses the 3rd check box. Hosting the vector tiles is done (although whether it's really "production-ready" is up for discussion).

**Comment by stevage** [comment]
Ok, have now done the ABS ones (CED_2018) too.

**Comment by stevage** [comment]
![Screenshot 2019-04-07 18 40 02](https://user-images.githubusercontent.com/678344/55681027-f1369a80-5964-11e9-85f2-e37b3f9d074d.png)
![Screenshot 2019-04-07 18 40 16](https://user-images.githubusercontent.com/678344/55681028-f1369a80-5964-11e9-9449-68d6ee9b84fa.png)

Here are the catalog items. Comments welcome on wording etc.

**Comment by AnaBelgun** [comment]
The description should be fine for now. Will let the AEC know once they are published and they can check them ;we can amend the text after.

**Comment by stevage** [comment]
Btw, regarding the csv-geo-au spec, I ended up with:

* com_elb_name_2019
* ced_code_2018
* ced_name_2018

Which is maybe a bit inconsistent re years. We don't really have a firm policy for identifying the year of datasets, but I called the AEC one "2019" because they're the boundaries as accurate for the 2019 election. The AEC doesn't actually put out a national dataset, so we can't follow their lead.

Meanwhile, the ABS clearly identifies their dataset as 2018, because the boundaries for this election were published in 2018 (and theirs follow suit).

Are we ok with this?

(I also realise that we don't actually need to update the csv-geo-au spec: it doesn't have any specifics about which years exist).
